src/main/java/org/jenkinsci/test/acceptance/po/JobsMixIn.java

```diff
@@ -58,7 +58,14 @@ public Object call() throws Exception {
     }
 
     public <T extends TopLevelItem> T get(Class<T> type, String name) {
-        return newInstance(type, injector, url("job/%s/", name), name);
+        try {
+            assert getContext() != null;
+            return newInstance(type, getContext(), url("job/%s/", name), name);
+        } catch (AssertionError e) {
```

Comment:
Fair enough. I propose a different solution with an explicit check for the constructor, please see new commit.